queries/ruby/locals.scm

```diff
@@ -36,7 +36,8 @@
 
 (module name: (constant) @definition.namespace)
 (class name: (constant) @definition.type)
-(method name: (identifier) @definition.function)
+(method name: [(identifier) (constant)] @definition.function)
+(singleton_method name: [(identifier) (constant)] @definition.function)
```

Comment:
So, despite their names, those are not actually methods ?

Reply:
@vigoux They are methods, they are just called "singleton methods". Other languages would call them class or static methods, but Ruby decided it wanted to be different.